include/tasks/wifi.h

```diff
@@ -36,8 +37,18 @@ struct wifi_client_cfg {
         char passwd[WIFI_PASSWD_MAX_LEN];
 };
 
+struct wifi_ap_cfg {
+        bool active;
+        char ssid[WIFI_SSID_MAX_LEN];
+        char password[WIFI_PASSWD_MAX_LEN];
+        size_t channel;
```

Comment:
should be uint8_t or uint32_t ? is size_t the appropriate type for describing the channel number?